cros_gralloc/gralloc4/CrosGralloc4Allocator.cc

```diff
@@ -46,19 +46,26 @@ Error CrosGralloc4Allocator::allocate(const BufferDescriptorInfo& descriptor, ui
         return Error::UNSUPPORTED;
     }
 
-    bool supported = mDriver->is_supported(&crosDescriptor);
-    if (!supported && (descriptor.usage & BufferUsage::COMPOSER_OVERLAY)) {
-        crosDescriptor.use_flags &= ~BO_USE_SCANOUT;
-        supported = mDriver->is_supported(&crosDescriptor);
-    }
+    /* If it is kmsro mode, skip checking the format & usage comb support
+     * In kmsro mode, the scanout buffer allocate through the non-3d methord,
+     * the virtio_dumb_bo_create function.
+     */
+    if (!mDriver->is_kmsro_enabled())
```

Comment:
We didn't check format & usage here, we can add one more in cros_gralloc_driver alloc function to capture issues?